src/components/FeaturedProjects.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import styled from 'styled-components'
+import { InnerWrapper, OuterWrapper, P, A } from 'StyledComponents/GlobalStyles'
+import { SectionHeading, SubHeading } from 'StyledComponents/HeadlineStyles'
+import { Tag, TagContainer } from 'StyledComponents/TagStyles'
+import data from '../assets/ProjectData.json'
+
+export const FeaturedProjects = () => {
+  return (
+    <OuterWrapper grey>
+      <InnerWrapperFeat>
+        <SectionHeading>FEATURED PROJECTS</SectionHeading>
+
+        <ProjectsWrapper>
+          {data.map((item) => (
```

Comment:
At the moment there are always 4 projects showing, even if you're on mobile where it should only be 2. The easiest way to solve it would probably be to just make another json-file where you only list 2 projects and when you're looking in mobile view you see mobile-json and when looking on desktop you see desktop-json.